### PR TITLE
[release-4.16] WINC-1309: Add konflux configuration files

### PR DESCRIPTION
**Pull request by sebsoto**
Adding configuration files to each release branch as instructed by https://konflux-ci.dev/docs/advanced-how-tos/managing-multiple-versions/

### CODE REVIEW

File: .tekton/windows-machine-config-operator-bundle-release-4-16-pull-request.yaml [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+  labels:
+    appstudio.openshift.io/application: windows-machine-config-operator-release-4-16
+    appstudio.openshift.io/component: windows-machine-config-operator-bundle-release-4-16
+    pipelines.appstudio.openshift.io/type: build
+  name: windows-machine-config-operator-bundle-release-4-16-on-pull-request
+  namespace: windows-machine-conf-tenant
+spec:
+  params:
+  - name: dockerfile
+    value: build/bundle-konflux.Dockerfile
```
Comment (jrvaldes) [inline .tekton/windows-machine-config-operator-bundle-release-4-16-pull-request.yaml:24]: @sebsoto, does this file exist in 4.16?

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

**Comment by sebsoto** [comment]
/override ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/wicd-unit-vsphere

**Comment by sebsoto** [comment]
Force merging as application isn't yet created

**Pull request merged**